tests/lts_compatibility.py

```diff
@@ -501,10 +503,28 @@ def run_ledger_compatibility_since_first(args, local_branch, use_snapshot):
                         version,
                     )
 
+                # We accept ledger chunk file differences during upgrades
+                # from 2.x-rc6 to 2.x-rc7 and from 1.x to 2.x-rc7. This
+                # is necessary because we broke the ledger compatibility between
+                # those versions (see https://github.com/microsoft/ccf/issues/3613).
```

Comment:
I think this is too strong of a statement. The ledger compatibility isn't broken between these versions, it's only that the ledger files may not be chunked at the same interval between these nodes since 2.x nodes require a ledger entry flag to create a new chunk, and this flag isn't present on 1.x ledgers.

Reply:
Oh yes that's what I meant to say, rephrasing.